### ISSUE TITLE
Windows invalid volume specification

### ISSUE THREAD

**Issue by moaead** [nginx-proxy/nginx-proxy#1471]
I am using windows containers and was wondering if ti can be possible to use nginx-proxy I did a lot of search on the closed issues but nothing worked.

I'v tried
$Env:COMPOSE_CONVERT_WINDOWS_PATHS=1
mapping to pipe.

But nothing worked. so If its not supported on windows containers can you please mention that in the README file.

Thanks for the hard work.

**Comment by tkw1536** [comment]
I don't think nginx-proxy can run as a windows container at the moment.

**Comment by moaead** [comment]
> I don't think nginx-proxy can run as a windows container at the moment.

Yeah seems so, well I got around it by using nginx on widows server core with custom powershell scripts to add proxies and all working great.
